tokio-console/src/view/task.rs

```diff
@@ -68,6 +68,17 @@ impl TaskView {
             })
             .collect();
 
+        let location_heading = "Location: ";
+        let max_width_stats_area = area.width - 45; //NOTE: 45 is min width needed, this is a calculated number according to the string: 'Last woken: 75.831727ms ago' but with some more extra pixels.
+        let location_lines_vector: Vec<String> = task
+            .location()
+            .to_string()
+            .chars()
+            .collect::<Vec<char>>()
+            .chunks(max_width_stats_area as usize)
+            .map(|chunk| chunk.iter().collect())
+            .collect();
+        // Id, Name, Target, Location (multiple), total, busy, scheduled, and idle times + top/bottom borders
```

Comment:
I think I wrote something confusing, it's include the task stats height as a variable here:
```suggestion
        // Id, Name, Target, Location (multiple), total, busy, scheduled, and idle times + top/bottom borders
        let task_stats_height = 9 + location_lines_vector.len() as u16;
```